src/components/subtitlesettings/subtitleappearancehelper.js

```diff
@@ -6,25 +6,34 @@
 function getTextStyles(settings, preview) {
     const list = [];
 
+    let scale = 1;
```

Comment:
Could we just use `devicePixelRatio` here?

https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio

Reply:
IME that tends to be hardcoded to a platform specific value that isn't the most helpful